src/components/Map.jsx

```diff
@@ -1,27 +1,74 @@
-import MapGl, { Layer, Source } from "react-map-gl/maplibre";
 import "maplibre-gl/dist/maplibre-gl.css";
+import T from "prop-types";
+import MapGl, { Layer, Source } from "react-map-gl/maplibre";
+import { useState, useEffect } from "react";
+import { AttributionControl } from "react-map-gl";
```

Comment:
This needs to imported from "react-map-gl/maplibre", otherwise we need to install MapboxGL. 

```suggestion
import MapGl, { Layer, Source, AttributionControl } from "react-map-gl/maplibre";
import { useState, useEffect } from "react";
```